docs/sources/flow/getting-started/migrating-from-static.md

```diff
@@ -0,0 +1,322 @@
+	job_name        = "local-agent"
+	scrape_interval = "15s"
+}
+
+prometheus.remote_write "metrics_test" {
+	endpoint {
+		name = "test-a653a1"
+		url  = "https://prometheus-us-central1.grafana.net/api/prom/push"
+
+		basic_auth {
+			username = "USERNAME"
+			password = "PASSWORD"
+		}
+
+		queue_config { }
+
+		metadata_config { }
+	}
+}
+
+local.file_match "logs_varlogs_varlogs" {
+	path_targets = [{
+		__address__ = "localhost",
+		__path__    = "/var/log/*.log",
+		host        = "mylocalhost",
+		job         = "varlogs",
+	}]
+}
+
+loki.process "logs_varlogs_varlogs" {
+	forward_to = [loki.write.logs_varlogs.receiver]
+
+	stage.match {
+		selector = "{filename=\"/var/log/*.log\"}"
+
+		stage.drop {
+			expression = "^[^0-9]{4}"
+		}
+
+		stage.regex {
+			expression = "^(?P<timestamp>\\d{4}/\\d{2}/\\d{2} \\d{2}:\\d{2}:\\d{2}) \\[(?P<level>[[:alpha:]]+)\\] (?:\\d+)\\#(?:\\d+): \\*(?:\\d+) (?P<message>.+)$"
+		}
+
+		stage.pack {
+			labels           = ["level"]
+			ingest_timestamp = false
+		}
+	}
+}
+
+loki.source.file "logs_varlogs_varlogs" {
+	targets    = local.file_match.logs_varlogs_varlogs.targets
+	forward_to = [loki.process.logs_varlogs_varlogs.receiver]
+
+	file_watch {
+		min_poll_frequency = "1s"
+		max_poll_frequency = "5s"
+	}
+}
+
+loki.write "logs_varlogs" {
+	endpoint {
+		url = "https://USER_ID:API_KEY@logs-prod3.grafana.net/loki/api/v1/push"
+	}
+	external_labels = {}
+}
+
+```
+
+## Limitations
+
+Configuration conversion is done on a best-effort basis. The Agent will issue
+warnings or errors where the conversion cannot be performed.
+
+Once the configuration is converted, we recommend that you review
+the Flow Mode configuration file created, and verify that it is correct
+before starting to use it in a production environment.
+
+Furthermore, we recommend that you review the following checklist:
+
+* The following configuration options are not available for conversion to flow
+  mode: [Integrations next], [Traces], and [Agent Management]. Any
+  additional unsupported features are returned as Error[s] during conversion.
+* There is no gRPC server to configure for flow mode, so any non-default config
+  will show as unsupported during the conversion.
+* Check if you are using any extra command line arguments with Static Mode which
+  are not present in your config file. For example, `-server.http.address`.
+* Check if you are using any environment variables in your [Static] config.
+  These will be evaluated during conversion and you may want to replace them
+  with the Flow Standard library [env] function after conversion.
+* Review additional [Prometheus Limitations] for limitations specific to your
+  [Metrics] config.
+* Review additional [Promtail Limitations] for limitations specific to your
+  [Logs] config.
+* Note that the logs produced by the Agent will differ from those
+  produced by Grafana Agent Static Mode.
+* Note that the Agent exposes the [Grafana Agent Flow UI][].
```

Comment:
```suggestion
* There is no gRPC server to configure for Flow mode, so any non-default config
  will show as unsupported during the conversion.
* Check if you are using any extra command line arguments with Static mode that
  are not present in your configuration file. For example, `-server.http.address`.
* Check if you are using any environment variables in your [Static] mode configuration.
  These will be evaluated during conversion and you may want to replace them
  with the Flow Standard library [env] function after conversion.
* Review additional [Prometheus Limitations] for limitations specific to your
  [Metrics] config.
* Review additional [Promtail Limitations] for limitations specific to your
  [Logs] config.
* The logs produced by Grafana Agent Flow mode will differ from those
  produced by Static mode.
* Grafana Agent exposes the [Grafana Agent Flow UI][].
```

Reply:
I'm not sure what happened on this one, maybe some conflicting commit suggestions?